__tests__/app.test.js

```diff
@@ -123,3 +123,69 @@ describe('/api/articles', () => {
 			});
 	});
 });
+
+describe('/api/articles/:article_id/comments', () => {
+	test('GET 200: responds with an array of comments with the associated article id', () => {
+		return request(app)
+			.get('/api/articles/3/comments')
+			.expect(200)
+			.then(({ body }) => {
+				const { comments } = body;
+				expect(comments).toEqual([
+					{
+						comment_id: 11,
+						body: 'Ambidextrous marsupial',
+						article_id: 3,
+						author: 'icellusedkars',
+						votes: 0,
+						created_at: expect.any(String),
+					},
+					{
+						comment_id: 10,
+						body: 'git push origin master',
+						article_id: 3,
+						author: 'icellusedkars',
+						votes: 0,
+						created_at: expect.any(String),
+					},
+				]);
+			});
+	});
+
+	test('GET 200: responds with an array of comments ordered by created_at in descending order', () => {
+		return request(app)
+			.get('/api/articles/1/comments')
+			.expect(200)
+			.then(({ body }) => {
+				const { comments } = body;
+				expect(comments).toBeSortedBy('created_at', { descending: true });
```

Comment:
not 100% but I'm not sure what would happen if the array of comments was empty and if this would still pass, so I would check the length just before this either way